golang/internal/codewars.go

```diff
@@ -0,0 +1,86 @@
+package internal
+
+import "sort"
+
+func Even_or_odd(x int) (answer string) {
+	if (x % 2) == 0 {
+		answer = "Even"
+	} else {
+		answer = "Odd"
+	}
+	return
+}
+
+func Sheep(sheep []bool) (answer int) {
+	answer = 0
+	for _, value := range sheep {
+		if value == true {
+			answer += 1
+		}
+	}
+	return
+}
+
+func Count_with_son(number int) (answer []int) {
```

Comment:
в принципе в го принят CamelCase
```suggestion
func CountWithSon(number int) (answer []int) {
```